### PR TITLE
Fix two bugs in master-setup.sh

### PR DESCRIPTION
**Pull request by jackgr**
Couple of bug fixes to make the script run correctly when it's run locally and when there's not already a cluster-insight replication controller running on the master.

### CODE REVIEW

File: install/master-setup.sh [hunk 1]
```diff
@@ -91,15 +91,20 @@ if [[ "${MASTER_ID}" != "" ]]; then
 fi
 
 # stop the Cluster-Insight minion collectors.
```
Comment (EranGabber) [inline install/master-setup.sh]: Please change the comment to:
# stop the Cluster-Insight replication controller (and its minions) if it is running.

File: install/master-setup.sh [hunk 2]
```diff
@@ -91,15 +91,20 @@ if [[ "${MASTER_ID}" != "" ]]; then
...
-echo "stop the Cluster-Insight replication controller and minions"
-exec_command kubectl stop -f ${REP_CONTROLLER}
+rc_output="$(kubectl get rc | fgrep ${IMAGE})"
+if [[ -n $rc_output ]] ; then
```
Comment (EranGabber) [inline install/master-setup.sh]: You have to be careful when rc_output is empty. You may get a syntax error if it is empty.
Please change the condition to:
if [[ -n "${rc_output}" ]] ; then

**Comment by EranGabber** [comment]
Please address my comments and ping me.

**Comment by jackgr** [comment]
I've addressed your comments. Note that quoting is not necessary for -n, since it deals correctly with unset and null variables by design, but I've quoted the expression, anyway. On the comment, I took it verbatim, except for the parenthetical expression "(and it minions)" since replication controllers don't have minions - they have pods. If you want to add that back in or further revise the comment, I'd suggest that you do it after this PR is merged.

File: install/master-setup.sh (hunk 1 above)
Comment (jackgr) [inline install/master-setup.sh]: Done. See comment on github.


Jack Greenfield | Google

**Pull request merged**